### PR TITLE
testing step-50 to step-59

### PR DESCRIPTION
**Pull request by vyushut**
Proceeding after https://github.com/dealii/dealii/pull/17100, https://github.com/dealii/dealii/pull/17084, https://github.com/dealii/dealii/pull/17130, #17165

### CODE REVIEW

File: examples/step-54/step-54.cc [hunk 1]
```diff
@@ -170,6 +170,7 @@ namespace Step54
   void TriangulationOnCAD::read_domain()
   {
     TopoDS_Shape bow_surface = OpenCASCADE::read_IGES(cad_file_name, 1e-3);
+    std::cout << " Read " << cad_file_name << std::endl;
```
Comment (tjhei) [inline examples/step-54/step-54.cc:173]:
```suggestion
    std::cout << "  Read " << cad_file_name << std::endl;
```
I would go with 2 spaces.

File: examples/step-54/step-54.cc [hunk 2]
```diff
@@ -350,9 +351,10 @@ namespace Step54
   {
     const std::string filename =
       (output_filename + "_" + Utilities::int_to_string(cycle) + ".vtk");
-    std::ofstream logfile(filename);
+    std::ofstream vtu_file(filename);
```
Comment (tjhei) [inline examples/step-54/step-54.cc:354]:
```suggestion
    std::ofstream vtk_file(filename);
```

File: examples/step-54/step-54.cc [hunk 3]
```diff
@@ -350,9 +351,10 @@ namespace Step54
...
     GridOut       grid_out;
-    grid_out.write_vtk(tria, logfile);
+    grid_out.write_vtk(tria, vtu_file);
+    std::cout << " Wrote " << filename << std::endl;
```
Comment (tjhei) [inline examples/step-54/step-54.cc:357]:
```suggestion
    std::cout << "  Wrote " << filename << std::endl;
```

**Review by tjhei — APPROVED** [review]

**Review by tjhei — APPROVED** [review]

**Comment by tjhei** [comment]
Thanks. Would you mind squashing your commits?

**Review by tjhei — APPROVED** [review]

**Comment by tjhei** [comment]
the github CI is not running here for some reason. @vyushut can you try to edit your commit message and push again? Maybe that will trigger a new build.

**Comment by bangerth** [comment]
It seems to have worked.

**Pull request merged**